sql/src/main/java/io/crate/planner/node/dql/join/NestedLoop.java

```diff
@@ -94,8 +90,8 @@
      * a | 3
      * b | 3
      */
-    public NestedLoop(PlannedAnalyzedRelation left,
-                      PlannedAnalyzedRelation right,
+    public NestedLoop(PlanAndPlannedAnalyzedRelation left,
```

Comment:
please keep using PlannedAnalyzedRelation,   PlanAndPlannedAnalyzedRelation is "implementation detail"